desktop/apps/auction2/reducers.js

```diff
@@ -1,21 +1,123 @@
 import { combineReducers } from 'redux'
 import u from 'updeep'
+import { contains } from 'underscore'
+import { data as sd } from 'sharify'
+import * as actions from './actions'
 
 const initialState = {
+  minEstimateRangeDisplay: 50,
+  maxEstimateRangeDisplay: 50000,
   artworks: [],
-  isListView: false
+  total: 0,
+  isListView: false,
+  isFetchingArtworks: false,
+  sortMap: {
+    "lot_number": "Lot Number (asc.)",
+    "-lot_number": "Lot Number (desc.)",
+    "-searchable_estimate": "Most Expensive",
+    "searchable_estimate": "Least Expensive"
+  },
+  aggregatedArtists: [],
+  aggregatedMediums: [],
+  filterParams: {
+    size: 50,
+    page: 1,
+    sale_id: sd.AUCTION.id,
+    aggregations: ['TOTAL', 'MEDIUM', 'FOLLOWED_ARTISTS', 'ARTIST'],
+    sort: 'lot_number',
+    gene_ids: [],
+    artist_ids: [],
+    estimate_range: '',
+    ranges: {
+      estimate_range: {
+        min: 50,
+        max: 50000
+      }
+    }
+  }
 }
 
 function auctionArtworks(state = initialState, action) {
```

Comment:
Thoughts on how this reducer is organized? (i.e. right now it's just one massive case statement... should it be broken out?)